desktop_version/src/Game.cpp

```diff
@@ -400,7 +400,7 @@ void Game::lifesequence(void)
         if (lifeseq > 5) gravitycontrol = savegc;
 
         lifeseq--;
-        if (INBOUNDS_VEC(i, obj.entities) && lifeseq <= 0)
+        if (INBOUNDS_VEC(i, obj.entities) && lifeseq <= 0 || noflashingmode)
```

Comment:
This line needs to have parentheses to visually disambiguate which conditions are being AND-ed and OR-ed.